src/main/java/org/infobourg/betterrss/models/RssFlux.kt

```diff
@@ -0,0 +1,19 @@
+package org.infobourg.betterrss.models
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+import java.time.LocalDateTime
+
+//@Document
+@Document(collection = "RssFlux")
+data class RssFlux(
+        @Id
+        var id: String="",
+        var idWorkspace: String="",
+        var idChannel: String="",
+        var idMessage: String="",
+        var idThread: String="",
```

Comment:
`idThread` should be nullable (`String?`):
- null when a feed is linked to a channel
- not-null when a feed is linked to a message
```suggestion
        var idThread: String? = null,
```

Reply:
How a feed can be linked to a channel? can you do a thread from a channel and not a message?